flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_AirportList.js

```diff
@@ -0,0 +1,77 @@
+const airportTypes = {
```

Comment:
We actually don't need airport types since Dep, Arr and Alt have their dedicated row.
It's just important to know whether it's managed or not.

Reply:
Replaced airportTypes with isManaged boolean